operator/pkg/reconciliation/constructor_test.go

```diff
@@ -211,7 +212,7 @@ func TestCassandraDatacenter_buildPodTemplateSpec_labels_merge(t *testing.T) {
 
 	expected := dc.GetRackLabels("testrack")
 	expected[api.CassNodeState] = stateReadyToStart
-	expected["app.kubernetes.io/managed-by"] = "cassandra-operator"
```

Comment:
I feel so much shame right now, but this is how name changes go - one leak and the costs add up later